Library/Homebrew/dev-cmd/linkage.rb

```diff
@@ -84,6 +88,21 @@ def display_normal_output
       display_items "Possible undeclared dependencies", @undeclared_deps
     end
 
+    def display_reverse_output
+      return if @reverse_links.empty?
+      puts "Links to:"
+      sorted = @reverse_links.sort
+      sorted.each do |dylib, files|
+        next if dylib == "/usr/lib/libSystem.B.dylib"
```

Comment:
I was looking at postgresql when I implemented this and it felt like just a whole page of executables to skip; I'm having trouble imagining caring that something is linking to libSystem but it seems y'all feel strongly so I can add it back.